Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 10
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	get_files
	1

[Sat Feb 27 00:02:44 2021]
rule get_files:
    output: results/analysis_files_list.csv
    jobid: 0

/home/hbrandao/anaconda3/bin/python /home/hbrandao/libs/data_analysis_Fbn2/snakemake/.snakemake/scripts/tmp64e_7wre.create_files_list.py
[Sat Feb 27 16:20:20 2021]
Error in rule get_files:
    jobid: 0
    output: results/analysis_files_list.csv

RuleException:
CalledProcessError in line 77 of /home/hbrandao/libs/data_analysis_Fbn2/snakemake/snakefile:
Command 'set -euo pipefail;  /home/hbrandao/anaconda3/bin/python /home/hbrandao/libs/data_analysis_Fbn2/snakemake/.snakemake/scripts/tmp64e_7wre.create_files_list.py' died with <Signals.SIGTERM: 15>.
  File "/home/hbrandao/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2347, in run_wrapper
  File "/home/hbrandao/libs/data_analysis_Fbn2/snakemake/snakefile", line 77, in __rule_get_files
  File "/home/hbrandao/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 568, in _callback
  File "/home/hbrandao/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/hbrandao/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 554, in cached_or_run
  File "/home/hbrandao/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2359, in run_wrapper
Will exit after finishing currently running jobs.
Shutting down, this might take some time.
Complete log: /home/hbrandao/libs/data_analysis_Fbn2/snakemake/.snakemake/log/2021-02-27T000244.572187.snakemake.log
